server/svix-server/src/core/cache.rs

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: © 2022 Svix Authors
+// SPDX-License-Identifier: MIT
+
+use enum_dispatch::enum_dispatch;
+
+use super::kv_backend::{memory::MemoryKv, none::NoKvBackend, redis::RedisKv, KvBackend};
```

Comment:
kv is a very confusing name, and the fact that it's called "MemoryKv" vs "KvBackend" and "NoKvBackend" is even more confusing. I'd at least prefix all of them with Kv if you must, though anyhow, as said, we need a different name.

Reply:
Maybe even a KeyValueStore a better name than kv (you can still use kv for the macro to keep it short).